ili9341/parallel_atsamd51.go

```diff
@@ -72,15 +80,52 @@ func (pd *parallelDriver) write16(data uint16) {
 
 //go:inline
 func (pd *parallelDriver) write16n(data uint16, n int) {
-	for i := 0; i < n; i++ {
-		pd.write8(byte(data >> 8))
-		pd.write8(byte(data))
+	if n <= 0 {
+		return
+	}
+
+	d0 := byte(data >> 8)
+	d1 := byte(data)
+
+	if d0 == d1 {
+		pd.write8(d0)
+		pd.wrx()
+		for i := 1; i < n; i++ {
+			pd.wrx()
+			pd.wrx()
+		}
+	} else {
+
+		for i := 0; i < n; i++ {
+			pd.write8(byte(data >> 8))
+			pd.write8(byte(data))
+		}
 	}
 }
 
 //go:inline
 func (pd *parallelDriver) write16sl(data []uint16) {
-	for i, c := 0, len(data); i < c; i++ {
+	c := len(data)
+	i := 0
+	for i = 0; i < c-8; i += 8 {
```

Comment:
why 2 bucles ?
You are doing less loops, but  more code, is there something else?

Reply:
I thought that loop unrolling would make it faster.
But the pyportal_boing demo is somehow slower.
https://en.wikipedia.org/wiki/Loop_unrolling